homework/remove-vowels/vowel.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include <iostream>
+#include <vector>
+#include <string>
+#include <algorithm>
```

Comment:
It is best practice to keep header files as light as it's possible. That means it's best to only include necessary files (iostream and algorithm are redundant here - it's best to move them to .cpp file)